pgwatch3_rpc_server/src/examples/csv_receiver.go

```diff
@@ -0,0 +1,87 @@
+package examples
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	. "pgwatch3_rpc_receiver/sinks"
+)
+
+type CSVReceiver struct {
+	FullPath string
+}
+
+/*
+* Structure for CSV storage:
+*   - Database Folder
+*       - Metric1.csv
+*       - Metric2.csv
+ */
+
+var isSyncSignalHandleActive = false
+
+func HandleSyncSignals(recv *Receiver) {
+	isSyncSignalHandleActive = true
+	// val := recv.GetSyncChannelContent()
+	// if val.OPR == "DELETE" {
+	// 	fmt.Print("DELETE METRIC: ", val.MetricName)
+	// }
+	// isSyncSignalHandleActive = false
```

Comment:
Commented out code looks very unprofessional. It's always better to remove it completely rather than comment out. At the end of the day you always have the history in git, so you can always check out how it was and should you resurrect the code

Reply:
Sure I'll get rid of this